slip_driver_node:
  ros__parameters:
    serial_port: "/dev/ttyUSB0"
    baudrate: 115200
    timeout: 0.01
    poll_interval: 0.01